### DISCUSSION TITLE
penmods支持有道词典笔S6吗？

### DISCUSSION THREAD

**Discussion by LiuStar2233** [PenUniverse/PenMods#222] [你问我答]
penmods支持有道词典笔S6吗？如果不支持，能开发一个吗？

**Comment by Redbeanw44602** [comment]
没设备，没时间，没兴趣

**Comment by LiuStar2233** [comment]
6










在 2024-04-12 00:10:56，"RedbeanW" ***@***.***> 写道：

没设备，没时间，没兴趣
